### PR TITLE
docs: improve addstu command user guide documentation

### PR DESCRIPTION
**Pull request by ForAeons**
image

Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/212
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/169
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/166
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/161
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/200
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/162

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -199,16 +199,36 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
...
+
+<box type="warning" light>
+
+**Note:** The application will not wrap the text in the various fields. By [international standards](https://en.wikipedia.org/wiki/E.164), telephone numbers should not exceed 15 digits. And long names and email addresses are extremely rare, which falls outside of normal use cases. Lastly, most students will have no more than 2 majors.
+</box>
+
 Examples:
 * `addstu n/John Doe nn/e1234567 p/98765432 e/johnd@example.com m/Computer Science`
-* `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com m/Mathematics p/1234567 t/club`
+* `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com m/Mathematics, Physics p/1234567 t/club`
```
Comment (Anant1902) [inline docs/UserGuide.md:231]: Just a quick change to make the nusnet valid
```suggestion
* `addstu n/Betsy Crowe nn/e0123456 t/friend e/betsycrowe@example.com m/Mathematics, Physics p/1234567 t/club`
```

**Comment by ForAeons** [comment]
Probably not a good idea to cram some of the info in addstu sections. I could move it to FAQ section after ethan's pr is merged

File: docs/UserGuide.md [hunk 2]
```diff
@@ -199,16 +199,36 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
...
 * `addstu n/Betsy Crowe nn/e01234567`
```
Comment (Anant1902) [inline docs/UserGuide.md:232]: ditto
```suggestion
* `addstu n/Betsy Crowe nn/e0123456`
```

**Review by Anant1902 — CHANGES_REQUESTED** [review]
Requesting some quick fixes!

**Comment by ForAeons** [comment]
> Requesting some quick fixes!

Thanks for the catch

**Review by Anant1902 — APPROVED** [review]
LTGM! Thanks for the fix.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -199,17 +199,37 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
...
+
 Examples:
 * `addstu n/John Doe nn/e1234567 p/98765432 e/johnd@example.com m/Computer Science`
-* `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com m/Mathematics p/1234567 t/club`
-* `addstu n/Betsy Crowe nn/e01234567`
+* `addstu n/Betsy Crowe nn/e0123456 t/friend e/betsycrowe@example.com m/Mathematics, Physics p/1234567 t/club`
+* `addstu n/Betsy Crowe nn/e0123456`
 
 <markdown class="d-print-none">---</markdown>
 <div style="page-break-after: always;"></div>
```
Comment (belligerentbeagle) [inline docs/UserGuide.md:235]: Can use `{{ newPage }}` instead

File: docs/UserGuide.md [hunk 4]
```diff
@@ -199,17 +199,37 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
...
+<box type="warning" light>
+
+**Note:** The application will not wrap the text in the various fields. By [international standards](https://en.wikipedia.org/wiki/E.164), telephone numbers should not exceed 15 digits. And long names and email addresses are extremely rare, which falls outside of normal use cases. Lastly, most students will have no more than 2 majors.
+</box>
+
 Examples:
 * `addstu n/John Doe nn/e1234567 p/98765432 e/johnd@example.com m/Computer Science`
-* `addstu n/Betsy Crowe nn/e01234567 t/friend e/betsycrowe@example.com m/Mathematics p/1234567 t/club`
-* `addstu n/Betsy Crowe nn/e01234567`
+* `addstu n/Betsy Crowe nn/e0123456 t/friend e/betsycrowe@example.com m/Mathematics, Physics p/1234567 t/club`
```
Comment (belligerentbeagle) [inline docs/UserGuide.md:231]: use capital E insetad? `nn/E`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -199,17 +199,37 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
...
+* `addstu n/Betsy Crowe nn/e0123456`
```
Comment (belligerentbeagle) [inline docs/UserGuide.md:232]: ditto

**Review by belligerentbeagle — APPROVED** [review]
LGTM! Just a couple optional nits.

**Pull request merged**